jupyter-notebooks/analytics/user-guide/1. Getting Started with the Planet Analytics API.ipynb

```diff
@@ -1,9 +1,25 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup CSS Styles\n",
+    "from styles.plnbstyles import style_nb\n",
+    "style_nb(\"./styles/styles.css\")\n"
```

Comment:
Is this custom styling really necessary? @cameronbronstein's analytics notebook cleanup intentionally removed much of this custom styling in June. https://github.com/planetlabs/notebooks/pull/131

Custom styles are discouraged in this repo. Please see the contributing guide for more details: https://github.com/planetlabs/notebooks/blob/master/CONTRIBUTING.md#style

Reply:
Orestis had it in his notebook, and it really does make the reading experience much better with the custom styling -- callout boxes are easier to read, warning boxes are much clearer, etc.

Reply:
I've removed the custom CSS styling and extraneous HTML elements.